app/views/chambers/peerage/candidates/_gallery.html.erb

```diff
@@ -1,26 +1,14 @@
-<div id="blueimp-gallery" class="blueimp-gallery">
-    <div class="slides"></div>
-    <h3 class="title"></h3>
-    <a class="prev">‹</a>
-    <a class="next">›</a>
-    <a class="close">×</a>
-    <a class="play-pause"></a>
-    <ol class="indicator"></ol>
-</div>
-
-<div id="gallery"> 
-<% images.each do |img| %>
- <%= link_to image_tag(img.image.url(:thumb), alt: img.created_at.strftime('%b %-d, %Y')), img.image.url, title:  img.created_at.strftime('%b %-d, %Y')%>
+<table class="table">
+<thead>
+<th>Document</th><th>Name</th><th>Description</th><th>Submission Date</th><th>Submitted By</th>
+</thead>
+<% documents.each do |doc| %>
```

Comment:
Consider adding a message that there are no documents and you can use the form to upload one, if there aren't yet any? It might be confusing to sometimes see the table and sometimes not.